### ISSUE TITLE
ideas

### ISSUE THREAD

**Issue by saru95** [ExtremeEvilCorp/we-need-a-project#1]
Discuss here any ideas anyone's got ?

**Comment by saru95** [comment]
to start with .. how about an anonymous chat service (where identity of sender and reciever are kept secret) ?

**Comment by tfhulbert** [comment]
Good idea, with some sort of matching formula or something or just completely random?

**Comment by saru95** [comment]
we can use an ID like system i guess . something like what BBM used in its days . And maybe we can assign certain monikers with those IDs instead of associating them with the actual name ? Just a shot in the dark .... gotta think think more .
